### PR TITLE
fix(curriculum): Rotate css property

### PR DESCRIPTION
**Pull request by j0pgrm**
#47489

### CODE REVIEW

**Comment by moT01** [comment]
PR templates come with a checklist - can you please add the template back to your the description here and complete the checklist @j0pgrm - Here it is:
```
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #XXXXX

<!-- Feel free to add any additional description of changes below this line -->
```

Also, add the issue number at the bottom if this PR closes the issue. Thanks.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ var SpaceShuttle = function(targetPlanet){
 var zeus = new SpaceShuttle('Jupiter');
 ```
 
-The `class` syntax simply replaces the `constructor` function creation:
+The `class` syntax simply replaces the keyword `function` with `construction`.
```
Comment (jeremylt) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md:24]: We specifically decided against this change I thought?

Comment (jeremylt) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md:24]: It has nothing to do with the linked issue and should be removed from this PR.

**Comment by Sboonny** [comment]
I have closed this, because the original issue is still labelled https://github.com/freeCodeCamp/freeCodeCamp/labels/status%3A%20discussing.

Have a good day and happy coding

**Pull request closed without merging**